public/app/features/logs/components/LogRow.tsx

```diff
@@ -273,8 +279,20 @@ class UnThemedLogRow extends PureComponent<Props, State> {
       return (
         <>
           <LogRowContextProvider row={row} getRowContext={getRowContext} logsSortOrder={logsSortOrder}>
-            {({ result, errors, hasMoreContextRows, updateLimit, logsSortOrder }) => {
-              return <>{this.renderLogRow(result, errors, hasMoreContextRows, updateLimit, logsSortOrder)}</>;
+            {({ result, errors, hasMoreContextRows, updateLimit, runContextQuery, logsSortOrder }) => {
+              return (
+                <>
+                  {this.renderLogRow(
+                    result,
+                    errors,
+                    hasMoreContextRows,
+                    updateLimit,
+                    logsSortOrder,
+                    this.props.getLogRowContextUi,
```

Comment:
i think it would be better if we de-structured this to be a bit more consistent.

Reply:
Fixed in df01fc56c10c93d66337cf0b71be2c7a55c1ec8b